.github/workflows/buildsCI.yaml

```diff
@@ -279,8 +209,10 @@ jobs:
            exit 0
         fi
 
-  Linux_static_cpp:
-    runs-on: ubuntu-18.04    
+
+  #--------------------------------------------------------------------------------
+  U20_SH_Py_DDS:  # Ubuntu 2020, Shared, Python, DDS
```

Comment:
Why do we build DDS with Ubuntu 20 and not 18?

Reply:
Because right now, it requires 20.
Besides, like I said elsewhere -- 18 is deprecated and I wanted at least one job running with 20.

Reply:
OK,